<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-06T09:30:30 MSK" name="Surefire suite" finished-at="2021-08-06T09:31:07 MSK" duration-ms="37120">
    <groups>
    </groups>
    <test started-at="2021-08-06T09:30:30 MSK" name="Surefire test" finished-at="2021-08-06T09:31:07 MSK" duration-ms="37120">
      <class name="test.UsingPricingCalculatorTest">
        <test-method is-config="true" signature="browserSetUp()[pri:0, instance:test.UsingPricingCalculatorTest@bccb269]" started-at="2021-08-06T09:30:30 MSK" name="browserSetUp" finished-at="2021-08-06T09:30:32 MSK" duration-ms="2764" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserSetUp -->
        <test-method signature="testTotalMonthlyCostDisplayedCorrectlyInEmail()[pri:0, instance:test.UsingPricingCalculatorTest@bccb269]" started-at="2021-08-06T09:30:32 MSK" name="testTotalMonthlyCostDisplayedCorrectlyInEmail" finished-at="2021-08-06T09:31:06 MSK" duration-ms="33307" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Estimate Cost from email not equals to calculator expected [null] but found [USD 978.37]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Estimate Cost from email not equals to calculator expected [null] but found [USD 978.37]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at test.UsingPricingCalculatorTest.testTotalMonthlyCostDisplayedCorrectlyInEmail(UsingPricingCalculatorTest.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTotalMonthlyCostDisplayedCorrectlyInEmail -->
        <test-method is-config="true" signature="exit()[pri:0, instance:test.UsingPricingCalculatorTest@bccb269]" started-at="2021-08-06T09:31:06 MSK" name="exit" finished-at="2021-08-06T09:31:07 MSK" duration-ms="724" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- exit -->
      </class> <!-- test.UsingPricingCalculatorTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
